graphql2/graphqlapp/destinationdisplayinfo.go

```diff
@@ -34,6 +34,11 @@ func (a *Destination) DisplayInfo(ctx context.Context, obj *graphql2.Destination
 func (a *Query) DestinationDisplayInfo(ctx context.Context, dest graphql2.DestinationInput) (*graphql2.DestinationDisplayInfo, error) {
 	app := (*App)(a)
 	cfg := config.FromContext(ctx)
+
+	if err := app.ValidateDestination(ctx, "input.action", &dest); err != nil {
```

Comment:
For this query the `input` variable is the `DestinationInput`

```suggestion
	if err := app.ValidateDestination(ctx, "input", &dest); err != nil {
```